### ISSUE TITLE
LOOT shows missing master when ran through MO2 only

### ISSUE THREAD

**Issue by cfs3rd** [TanninOne/modorganizer#443]
When I run LOOT through MO for Fallout 4 it shows that some plugins are missing masters even though the files are present. Put in a report at the LOOT GitHub here https://github.com/loot/loot/issues/519 . This only happens with MO. With NMM and manual installs LOOT works fine. If any other information is needed I will provide it.

Windows 10 64 bit Home latest build
 MO v 2.0.0a
 LOOT API v0.8.1-38-g7c4fe9f (64 bit)

**Comment by Wadu436** [comment]
I have the same problem: http://puu.sh/lZbvC/bb3025f073.png
Windows 10 Home 64-bit
MO2 Alpha 2
LOOT V0.8.1-g7c4fe9f (64bit)

**Comment by JourneyOver** [comment]
Hell you guys are lucky, I can't even get loot to show more than just the fallout4.esm file when running through MO :/

**Comment by Wadu436** [comment]
@JourneyOver You have to use the 64-bits version of LOOT. It's on the FO4 Nexus under old versions. The number is LOOT v0.8.1-38-g7c4fe9f

**Comment by GSDFan** [comment]
I think I may have found some useful info on this. If I rename or remove the plugins.txt from Username\AppData\Local\Fallout4 the warning messages about missing ESPs will go away. 

Can someone confirm this?

I also have the correct number of ESPs as when I had plugins.txt in AppData folder.

**Comment by GSDFan** [comment]
Just and update to my previous post, with plugins.txt removed from AppData and LOOT is run and has sorted the ESPs amd ESMs, upon LOOT exit all the ESPs and ESMs will become unchecked.

**Comment by Wadu436** [comment]
If you copy the plugins.txt from your MO Profile folder to Username/AppData/Local/Fallout4 and then do LOOT, the plugins won't be disabled

**Comment by GSDFan** [comment]
Yes, I know that. It also needs to be there in order for LOOT to be able to sort properly from what I see. I did not see LOOT sort at all without it there. LOOT appears to see my full list of plugins when it is not there. I can switch profiles, disable all mods in a profile so that fallout4.esm is the only active item and when run it will see the full list of mods I had when I first ran LOOT.

**Comment by JourneyOver** [comment]
@Wadu436 I do run the 64-bit version of loot, I had it working once before but then all of a sudden it just randomly stopped working and hasn't worked correctly since even if I remove all of the loot stuff and anything else that could be conflicting with it.

**Comment by GSDFan** [comment]
On the sorting thing, I should say it was in LOOT that I saw the sort happen. When you exit the program I did not see the changes in MO right pane.

**Comment by Wadu436** [comment]
@GSDFan Just after starting LOOT, unlock MO. Then after applying in LOOT, switch to the data tab and back to plugins

**Comment by GSDFan** [comment]
Thanks, Unlocking appears to be the key. It appears that MO is not doing an auto refresh when locked and LOOT is exited. I tried doing a refresh from the pull down and switching tabs after loot is run and the load order will not change. When I run LOOT, unlock MO, sort and then then return to MO after exiting LOOT and do a refresh or switch tabs the right pane load order will change.
